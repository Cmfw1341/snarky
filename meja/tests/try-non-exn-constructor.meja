type t = A;

let x = fun () => {
  try (failwith("")) {
    | A => 15
  };
};
