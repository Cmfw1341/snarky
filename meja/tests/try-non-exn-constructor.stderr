File "tests/try-non-exn-constructor.meja", line 5, characters 6-7:
Error: Incompatible types exn and t:
       Cannot unify exn and t
